packages/trader/src/Modules/MT5/Containers/mt5-password-manager-modal.jsx

```diff
@@ -142,96 +143,36 @@ const MT5PasswordManagerTabContentWrapper = ({ multi_step_ref, steps }) => (
-                                        {...field}
-                                        autoComplete='new-password'
-                                        label={localize('New password')}
-                                        hint={
-                                            !has_warning &&
-                                            localize(
-                                                'Strong passwords contain at least 8 characters, combine uppercase and lowercase letters and numbers.'
-                                            )
-                                        }
-                                        error={touched.new_password && errors.new_password}
-                                        onChange={e => {
-                                            setFieldTouched('new_password', true, true);
-                                            field.onChange(e);
-                                        }}
-                                        className='mt5-password-manager__new-password'
-                                        required
-                                    />
-                                )}
-                            </PasswordMeter>
-                        )}
-                    </Field>
-                    <div className='mt5-password-manager__actions'>
-                        <Button
-                            className='mt5-password-manager--button'
-                            is_disabled={
-                                isSubmitting ||
-                                !values.old_password ||
-                                !values.new_password ||
-                                Object.keys(errors).length > 0
-                            }
-                            is_loading={isSubmitting}
-                            text={localize('Change password')}
-                            primary
-                            large
-                        />
-                        <Button
-                            className='mt5-password-manager--button'
-                            type='button'
-                            onClick={() => {
-                                setPasswordType('main');
-                                multi_step_ref.current?.goNextStep();
-                            }}
-                            text={localize('Reset main password')}
-                            tertiary
-                            large
-                        />
-                    </div>
-                </Form>
-            )}
-        </Formik>
+        <div className='mt5-password-manager__trading-password-wrapper'>
+            <Icon icon='IcMt5OnePassword' size='128' />
+            <Text as='p' align='center' size='s' weight='bold'>
+                {is_existing_user && <Localize i18n_default_text='Set your trading password' />}
+                {!is_existing_user && <Localize i18n_default_text='All you’ll need from now is trading password' />}
```

Comment:
This sentence is incorrect.

Reply:
I'll fix it once copy is ready for this.